contracts/libraries/ConstraintsLib.sol

```diff
@@ -14,6 +14,8 @@ library ConstraintsLib {
     uint256 private constant _SERIES_MASK = type(uint8).max;
     uint256 private constant _NO_PARIAL_FILLS_FLAG = 1 << 255;
     uint256 private constant _NO_IMPROVE_RATE_FLAG = 1 << 254;
+    uint256 private constant _PRE_INTERACTION_CALL_FLAG = 1 << 253;
+    uint256 private constant _POST_INTERACTION_CALL_FLAG = 1 << 252;
```

Comment:
Not sure if its necessary though. If user signed order, which includes interaction data then its itself means that it requires interaction call. Processing order without this interaction data will ofc fail.

Reply:
Yes, but we need this flags to indicate if order creator want/expect this calls to happen.